modules/remotecontrol/Receiver.js

```diff
@@ -0,0 +1,47 @@
+/* global APP, JitsiMeetJS */
+const ConferenceEvents = JitsiMeetJS.events.conference;
+
+/**
+ * This class represents the receiver party for a remote controller session.
+ * It handles "remote-control-event" events and sends them to the
+ * API module. From there the events can be received from wrapper application
+ * and executed.
+ */
+class Receiver {
```

Comment:
Actually Receiver class is kind of static for the whole app, so I can live with that.